## Request Constants
uint8 CIRCLE = 1
uint8 SEMICIRCLE = 2
uint8 QUARTER_CIRCLE = 3
uint8 TRIANGLE = 4
uint8 SQUARE = 5
uint8 RECTANGLE = 6
uint8 TRAPEZOID = 7
uint8 PENTAGON = 8
uint16 HEXAGON = 9
uint16 HEPTAGON = 10
uint16 OCTAGON = 11
uint16 STAR = 12
uint16 CROSS = 13


# Request Fields

uint16 shape